index.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<!--[if lt IE 7]>      <html class="no-js lt-ie9 lt-ie8 lt-ie7"> <![endif]-->
+<!--[if IE 7]>         <html class="no-js lt-ie9 lt-ie8"> <![endif]-->
+<!--[if IE 8]>         <html class="no-js lt-ie9"> <![endif]-->
+<!--[if gt IE 8]>      <html class="no-js"> <!--<![endif]-->
+<html>
+    <head>
+        <meta charset="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <title>youtube</title>
+        <meta name="description" content="">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link rel="stylesheet" href="youtube.css">
+        <link rel="icon"  href="youtube.png">
+    </head>
+    <body>
+     
+        <div id="nav-bar">
+            <div id="nav-menu"><img src="menu.png"/></div>
+            <div id="logo">
+                <div id="logo1"><img  src="youtube.png"/></div>
+                <div id="logo2">YouTube</div>
+            </div>
+            <div id="nav-search">
+                <input type="search" placeholder="search"/><img src="loupe.png"/>
+            </div>
+        </div>
+    <div id="container">
+        
+        <div id="displyed_video_container">
+            <iframe width="560" height="315" src="https://www.youtube.com/embed/lJIrF4YjHfQ" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+            <h3>title of this video </h3>
+           
+        
+        </div> 
+        <div id="side-bar"> 
+            
+            <div class="side-bar-playliste">
+                
+                <img src="thumbnails/1.jpg"/>
+                <div> 
+                <p>title of video 1</p>
+                <p>2M views 2years ago </p>
+                </div>
+            </div>  
+            <div class="side-bar-playliste">
+                <img src="thumbnails/2.jpg"/>
+                <div> 
+                <p>title of video 1</p>
+                <p>2M views 2years ago </p>
+                </div>
+            </div>
+            <div class="side-bar-playliste">     
+                <img src="thumbnails/3.jpg"/>
+                <div> 
+                <p>title of video 1</p>
+                <p>2M views 2years ago </p>
+                </div>
+            </div>
+            <div class="side-bar-playliste">
+                <img src="thumbnails/4.jpg"/>
+                <div> 
+                <p>title of video 1</p>
+                <p>2M views 2years ago </p>
+                </div>
+            </div>
+            <div class="side-bar-playliste">
+                <img src="thumbnails/5.jpg"/>
+                <div> 
+                <p>title of video 1</p>
+                <p>2M views 2years ago </p>
+                </div>
+            </div>    
+            </div>
+           </div>   
+    </div>
+        <script src="" async defer></script>
+    </body>
+</html>
```

Comment:
please try to correct linters errors, code without linters errors is more clean and clear and looks professional

Reply:
[OPTIONAL]: The reason why we use Linter: A linter is a tool that analyzes your source code to flag programming errors, bugs, stylistic errors, and suspicious constructs(source: [Wikipedia](https://en.wikipedia.org/wiki/Lint_(software))).